tests/e2e/page-objects/Layout.js

```diff
@@ -23,8 +23,15 @@ export default class LayoutPage {
       .expect(bodyText)
       .notContains('Liquid Error')
       .expect(bodyText)
-      .notContains('RenderFormTag Error')
+      .notContains('RenderFormTag Error:')
       .expect(bodyText)
-      .notContains('QueryGraphTag Error');
+      .notContains('QueryGraphTag Error:')
+      .expect(bodyText)
+      .notContains('ExecuteQueryTagError:');
+  }
+
+  async checkURL(domain) {
+    const getPageHref = ClientFunction(() => document.location.href.toString());
```

Comment:
Maybe it would be easier to just click the link - that would check if link is working at the same time, instead of programmatically changing the url.

Reply:
Fixed